src/data/agile-quiz.ts

```diff
@@ -279,6 +279,16 @@ const agileQuiz = [
       "Feature Driven Development (FDD) is the practice of incrementally developing and frequently releasing new features for software products.",
     Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/"
   },
+  {
+    Question: "In Agile, Which agile methodology has no defined roles or functions and each member can assume the available tasks?",
+    Answer: "Kanban",
+    Distractor1: "Scrum",
+    Distractor2: "Lean",
+    Distractor3: "Agile Inception ",
+    Explanation:
+      "The Kanban method does not prescribe roles or meetings as do other frameworks such as Scrum. However, roles and meetings have been identified from experience in organizations that are relevant for the organization to follow the principles of the method through its practices..",
```

Comment:
I think just having this first sentence here is fine

```suggestion
      "The Kanban method does not prescribe roles or meetings unlike other frameworks such as Scrum.",
```